src/vendor/signals/stripe_signals.py

```diff
@@ -72,11 +70,13 @@ def stripe_delete_customer_signal(sender, instance, **kwargs):
 
     processor = StripeProcessor(instance.site)
 
+    post_delete.disconnect(sender=CustomerProfile, dispatch_uid="stripe_customer_post_delete")
```

Comment:
Now I might be completely misunderstanding this, but what happens if another profile is delete _while_ waiting on a response from stripe_delete_object

Reply:
yeah I need to find a better way to solve that scenario